src/content/admin/cloud-credentials/index.mdx

```diff
@@ -23,3 +23,9 @@ Follow our guides below to learn how to configure your Cloud Credentials for sup
 - [Amazon Web Services](admin/cloud-credentials/aws.mdx)
 - [Google Cloud](admin/cloud-credentials/gcp.mdx)
 
+## Manage Cloud Credentials using CRDs
```

Comment:
@codyjlandstrom i was doubting if adding this here or in each AWS, GCP credential description 🤔

Reply:
I went ahead and added it. I think it makes sense to let people know that it's an option